doc/developer/debugging.rst

```diff
@@ -61,6 +61,20 @@ With the default configuration in the file ``docker-compose.override.yaml``, you
 available in ``https``, and the authentication will work.
 
 
+Auto login
+----------
+
+To be automatically logged in with a user present in the database you should set in the ``vars.yaml``:
+
+.. code:: yaml
+
+  vars:
+    authentication:
+      dev_autologin: <username>
+
+Then you will directly be logged in to the user.
```

Comment:
It might be better to put this in development.ini file, to be sure this is never used on production.

Or add a message that this should never be committed.

Reply:
Or we could use an environment variable (that will not be present on production)

Reply:
+1 -> environment variable :-)

Reply:
So this could go directly to CONST_vars maybe.